### DISCUSSION TITLE
播放时显示时间和双击全屏失效

### DISCUSSION THREAD

**Discussion by Alexius66** [the1812/Bilibili-Evolved#4300] [综合讨论 / General]
不确定是不是bug，所以来这里问一下有没有人遇到相同情况，浏览器为edge 115.0.1901.170 ，播放器版本为4.2.1-rc.1903.0-64ec13f4

**Comment by Alexius66** [comment]
![image](https://github.com/the1812/Bilibili-Evolved/assets/73510996/db6a1859-a632-4156-90ce-4e18c4837663)
全新的bug！

**Reply by rookkkie** [reply]
双击全屏确实失效了

**Reply by Geeyun-JY3** [reply]
集中至 #4303
